server/routes/api/choropleth.py

```diff
@@ -35,15 +35,51 @@
   url_prefix='/tools/choropleth'
 )
 
-@bp.route('/api')
-def choropleth_api():
+@bp.route('/values')
+def choropleth_values():
     """Returns data for geographic subregions for a certain statistical 
             variable.
 
-    API Params: #TODO(iancostello): Split API calls up or move to client.
-        statVar -> The statistical variable to download, as a string.
-        perCapita -> Whether to return the per-capita value, as a string of a 
-                boolean.
+    API Params:
+        geoId -> The currently viewed geography to render, as a string.
+        level -> The subgeographic level to pull and display information for,
+                as a string. Choices: Country, AdministrativeLevel[1/2], City.
+        statVar -> The statistical variable to pull data about.
+
+    API Returns:
+        values -> statistical variable values for all subgeos.
+    """
+    # Get required request parameters.
+    requested_geoId = request.args.get("geoId")
+    if requested_geoId is None:
+        return jsonify({"error": "Must provide a 'geoId' field!"}, 400)
+    stat_var = request.args.get("statVar")
+    if requested_geoId is None:
+        return jsonify({"error": "Must provide a 'geoId' field!"}, 400)
+    display_level = get_sublevel(requested_geoId, request.args.get("level"))
+
+    # Get all subgeos.
+    geos_contained_in_place = dc.get_places_in(
+            [requested_geoId], display_level)[requested_geoId]
+    values_by_geo = dc.get_stats(geos_contained_in_place, stat_var)
+
```

Comment:
```suggestion
TODO(iancostello): Check for no values or too few values.
```

Reply:
I don't think we have to explicitly check here